app/admin/councillor_contributions.rb

```diff
@@ -15,14 +16,18 @@
       row :source
     end
 
-    h3 "Suggested Councillors"
-    table_for resource.suggested_councillors, class: "index_table" do
-      column :name
-      column :email
-    end
+  h3 "Suggested Councillors"
+  table_for resource.suggested_councillors, class: "index_table" do
+    column :name
+    column :email
   end
+end
 
-  action_item :download,:only => [:show] do
+  action_item :download, only: [:show] do
     link_to "Download the suggested councillors CSV", authority_councillor_contribution_path(resource.authority, resource.id, format: :csv)
   end
+  action_item :mark_as_reviewed, only: [:show] do
```

Comment:
While this is a nice straight-forward implementation @hisayohorie I think we should follow the pattern used in admin/authorities, and use the Active Admin DSL for adding controller actions. That way we save adding a route, and controller action for this small feature.

Checkout https://github.com/openaustralia/planningalerts/blob/master/app/admin/authorities.rb#L88-L98 and https://activeadmin.info/8-custom-actions.html .

It's a bit magical but basically use declare the controller action as a `member_action` in the actual admin file.

And then rather than having a link you get `button_to` which is more semantic. Active Admin creates the appropriate route by itself, because https://github.com/openaustralia/planningalerts/blob/master/config/routes.rb#L26 .